File: src/primeGenerator.py
import math
t = int(input())
for _ in range(t):
    a, b = map(int, input().split())
    for i in range(a, b+1):
        flag = False
        for j in range(2, int(math.sqrt(i))+1):
            if i % j == 0:
                flag = True
                break
        if not flag and i != 1:
            print(i)
    print()

File: src/primeGenerator.cpp

#include <iostream>
#include <cmath>

using namespace std;
int main(){
    int t;
    cin >> t;
    for(int i = 0; i < t; i++){
        int a, b;
        cin >> a >> b;
        for(int j = a; j <= b; j++){
            bool flag = false;
            for(int k = 2; k <= sqrt(j); k++){
                if(j % k == 0){
                    flag = true;
                    break;
                }
            }
            if(!flag && j != 1){
                cout << j << "\n";
            }
        }
        cout << "\n";
    }
}